arm-sql/2014-04-01/swagger/auditingPolicies.json

```diff
@@ -0,0 +1,410 @@
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ServerAuditingPolicy"
+                        },
+                        "description": "The required parameters for createing or updating a Azure SQL Server auditing policy."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ServerAuditingPolicy"
+                        }
+                    },
+                    "201": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ServerAuditingPolicy"
+                        }
+                    }
+                }
+            },
+            "get": {
+                "tags": [
+                    "AuditingPolicy"
+                ],
+                "operationId": "AuditingPolicy_GetServerPolicy",
+                "description": "Returns an Azure SQL Server auditing policy.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ServerAuditingPolicy"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "DatabaseAuditingPolicyProperties": {
+            "properties": {
+                "useServerDefault": {
+                    "type": "string",
+                    "description": "Indicates whether the default server auditing policy is used for that Azure SQL Database."
+                },
+                "auditingState": {
+                    "type": "string",
+                    "description": "Gets the auditing state of the Azure SQL Server auditing policy."
```

Comment:
Consider: I would suggest leaving out the Gets... part of the description since this is a data/model class. As such, it doesn't really get or set any properties, it represents what the property value currently is. This is similar to the fact that I rarely need separate descriptions for the getter and setter of a given property for a POCO class - I need to know what the property represents. 

The phrasing for useServerDefault sounds more natural to me.